scripts/check-third-party-license-file-paths

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Sorry, but in this case it works as expected.

From `checkbashisms` help:
```
This script performs basic checks for the presence of bashisms
in /bin/sh scripts and the lack of bashisms in /bin/bash ones.
```

The new script uses **no** bashisms, so there is **no** need to depend on bash. It should work with any POSIX shell, so we can use `/bin/sh` here.

```suggestion
#!/bin/sh
```

Also it looks like this script itself generates some false positive files:
```
../documentation 0
../and/or 0
../other 0
../materials 0
../provided 0
../with 0
../the 0
../distribution. 0
```

Reply:
Thank you, pushed the changes